primitives/arithmetic/src/per_things.rs

```diff
@@ -159,39 +161,22 @@ macro_rules! implement_per_thing {
 			}
 		}
 
-		/// Implement const functions
-		impl $name {
-			/// From an explicitly defined number of parts per maximum of the type.
-			///
-			/// This can be called at compile time.
-			pub const fn from_parts(parts: $type) -> Self {
```

Comment:
The idea was that this is a const function

Reply:
Yep I see that now